docs/src/rsh/compute/index.md

```diff
@@ -1467,8 +1467,6 @@ or `default` if the variant is `Right`.
  `{!rsh} fromRight(e, default)` is a convenience method that returns the value in `Right`,
 or `default` if the variant is `Left`.
 
-See following example summarizes all that have been said above:
```

Comment:
@EBEREGIT, we discussed this line in our morning standup. Please make the revisions per our conversation.